### PR TITLE
Add unit tests for ReadInsulinSensitivities.

### PR DESCRIPTION
**Pull request by moomoobloo**
- Harnessed some existing responses as unit tests.  
- Minor refactoring to improve testability.

Should help with #5 and #6.

### CODE REVIEW

File: decocare/commands.py [hunk 1]
```diff
@@ -1037,9 +1053,9 @@ def getData (self):
         break
       if units == 2:
         sensitivity = sensitivity / 10.0
-      schedule.append(dict(x=x, i=i, start=lib.basal_time(i), offset=i*30, sensitivity=sensitivity))
+      schedule.append(dict(x=x, i=i, start=str(lib.basal_time(i)), offset=i*30, sensitivity=sensitivity))
     labels = { True: 'Fast', False: 'Regular' }
-    return dict(sensitivities=schedule, first=self.data[0], units=self.UNITS.get(units))
+    return dict(sensitivities=schedule, first=isFast, units=ReadInsulinSensitivities.UNITS.get(units))
```
Comment (moomoobloo) [inline decocare/commands.py]: I wasn't sure, is `first` a typo for `fast`?

**Comment by scottleibrand** [comment]
I like tests and refactoring, but unfortunately I don't know Python well enough to really review this.

**Comment by moomoobloo** [comment]
Thanks, @scottleibrand. Maybe some past contributors can help. @loudnate @channemann .

**Comment by bewest** [comment]
Let's keep medtronic in the loop on all issues like this.

**Pull request merged**